packages/commonwealth/client/scripts/views/pages/CreateCommunity/steps/BasicInformationStep/BasicInformationForm/constants.ts

```diff
@@ -46,10 +47,9 @@ const chainIdsWithStakeEnabled = Object.values(
 ).map((c) => c.chainId);
 
 // Get chain id's from the app.config.chains for all eth and cosmos chains
-export const chainTypes = app.config.nodes
-  .getAll()
-  .filter(particularChainNodes)
-  .map((chain) => ({
+export const chainTypes = fetchCachedNodes()
```

Comment:
can we add a fallback value `{curent_logic} || [ ]`? since we will use it in multiple places (via exports)